v2.5/getting-started/openstack/installation/redhat.md

```diff
@@ -71,11 +71,11 @@ Configure the Calico repository:
     cat > /etc/yum.repos.d/calico.repo <<EOF
     [calico]
     name=Calico Repository
-    baseurl=http://binaries.projectcalico.org/rpm/{{ ppa_repo_name }}/
+    baseurl=https://binaries.projectcalico.org/rpm/{{ ppa_repo_name }}/
```

Comment:
Just spotted that you made this change in the `/v2.5/` directory which is the fork of our docs for the v2.5 release.  Please can you copy it to `/master/`?  (We maintain all the versions in one branch of the repo so we can easily back-apply changes.)

Reply:
Let me look into this and fix this one